wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -59,9 +59,9 @@
 static int mAtcaInitDone = 0;
 
 /* ATECC slotId handling */
-static atmel_slot_alloc_cb mSlotAlloc;
-static atmel_slot_dealloc_cb mSlotDealloc;
-static byte mSlotList[ATECC_MAX_SLOT];
+static atmel_slot_alloc_cb mSlotAlloc = NULL;
```

Comment:
Static variables are supposed to be zero initialized at startup. We can add a NULL here, but there are other places in our sources and many other projects where statics are assumed to be zero initialized. This is per the C89 standard section 3.5.7.